app/resources/forms.js

```diff
@@ -284,6 +284,22 @@ function validate_fields() {
   }
   hide($('mandatory_field_missing'));
 
+  //Validate email-id
+  var email_field = $('author_email');
+  var email_id = $('author_email').value;
+  if (email_id !== "")
+  {
+     var emailfilter=/^\w+[\+\.\w-]*@([\w-]+\.)*\w+[\w-]*\.([a-z]{2,4}|\d+)$/i;
```

Comment:
Let's use the same regex as the server-side:
https://github.com/google/personfinder/blob/master/app/subscribe.py#L26